infra/charts/management-service/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "0.0.1"
+appVersion: v0.6.0
 description: A Helm chart for Kubernetes Deployment of the XP Managment Service
 name: xp-management
-version: 0.0.1-dev0.1
+version: 0.6.0
```

Comment:
Since this is the Helm chart version, it need not match the app version (and doesn't really make sense to do so). It can just be `0.1.0`. The same comment applies to the treatment service.

Reply:
Okay, do we want to publish these changes? If so, perhaps `0.6.1` would make more sense.

Reply:
We only need to publish the chart for this PR - no changes to the app itself, correct? So `0.6.0` is fine.